Client/src/Components/Heading/index.jsx

```diff
@@ -0,0 +1,29 @@
+import { Typography } from "@mui/material";
+import PropTypes from "prop-types";
+
+/**
+ * A heading component that renders text with a specific heading level.
+ *
+ * @param {Object} props - The properties passed to the Heading component.
+ * @param {('h1'|'h2'|'h3')} props.component - The heading level for the component.
+ * @param {string} props.children - The content to display inside the heading.
+ * @returns {JSX.Element} The Typography component with specified heading properties.
+ */
+
+function Heading({ component, children }) {
+	return (
+		<Typography
+			component={component}
+			variant="h2"
+			fontWeight={600}
+		>
+			{children}
+		</Typography>
+	);
+}
+
+Heading.propTypes = {
+	component: PropTypes.oneOf(["h1", "h2", "h3"]).isRequired,
```

Comment:
Should we expand this to include other header levels?  I don't see a particular reason to limit to h1, h2, and h3 :thinking:

Reply:
I stopped at h3 cause I rarely see the need for lower headings (h4, 5 and 6), and that would be a way of adding more friction to other developers add those. If there comes the need we could add them here